api/v1beta1/azureclusteridentity_types.go

```diff
@@ -77,6 +77,11 @@ type AzureClusterIdentityStatus struct {
 }
 
 // +kubebuilder:object:root=true
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels.cluster\\.x-k8s\\.io/cluster-name",description="Cluster to which this AzureClusterIdentity belongs"
```

Comment:
Where is this label getting set? I don't see it when trying out this PR locally

```
k get azureclusteridentities.infrastructure.cluster.x-k8s.io
NAME               CLUSTER   READY   SEVERITY   REASON   TYPE               AGE
cluster-identity                                         ServicePrincipal   9m27s
```

Reply:
I don't think it ever does. And I'm not sure it would make sense to add it since an AzureClusterIdentity can be shared across multiple clusters, so I'd vote to remove this.